### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by 592363789**

### CODE REVIEW

**Comment by qiu-siqi** [comment]
Remove irrelevant information in your PPP?

File: docs/team/shijin.adoc [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+---
+
+== Overview
+
+Bibliotek is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+
+== Summary of contributions
+
+* **Mark a book as read, change the priority of a book and rate books** (major)
+** Added attributes rating, status, priority.
```
Comment (takuyakanbr) [inline docs/team/shijin.adoc]: Remove status and priority.

File: docs/team/shijin.adoc [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+* **Encrypt data with a password** (minor)
+** Add an `encrypt` command to encrypt data with a password.
+** This allows users to keep their data private and safe.
+
+* *Code contributed*: [https://github.com/CS2103JAN2018-F14-B4/main/tree/master/collated/functional.592363789.md[Functional code]] [https://github.com/CS2103JAN2018-F14-B4/main/tree/master/collated/test.592363789.md[Test code]] _{give links to collated code files}_
+
+* *Other contributions*:
+
+** Project management:
+*** Managed releases `v1.3` - `v1.5rc` (3 releases) on GitHub
```
Comment (takuyakanbr) [inline docs/team/shijin.adoc]: Remove.

File: docs/team/shijin.adoc [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+** Enhancements to existing features:
+*** Add the rating book command.
+*** Add the status and priority attributes.
```
Comment (takuyakanbr) [inline docs/team/shijin.adoc]: Remove.

File: docs/team/shijin.adoc [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+*** Add in Edit book command (Pull requests https://github.com/CS2103JAN2018-F14-B4/main/pull/111[#111]).
+*** Add in encrypt command.
+*** Add in decrypt and setkey command (Pull requests https://github.com/CS2103JAN2018-F14-B4/main/pull/134[#134])
+** Documentation:
+*** Did cosmetic tweaks to existing contents of the User Guide: https://github.com[#14]
+** Community:
+*** PRs reviewed : https://github.com[#12], https://github.com/CS2103JAN2018-F14-B4/main/pull/84[#84], https://github.com/CS2103JAN2018-F14-B4/main/pull/79[#79], https://github.com/CS2103JAN2018-F14-B4/main/pull/68[#68], https://github.com/CS2103JAN2018-F14-B4/main/pull/44[#44]
+** Tools:
+*** Integrated a third party library (Natty) to the project (https://github.com[#42])
+*** Integrated a new Github plugin (CircleCI) to the team repo
```
Comment (takuyakanbr) [inline docs/team/shijin.adoc]: Remove.

File: docs/team/shijin.adoc [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+*** Add the status and priority attributes.
+*** Add in Edit book command (Pull requests https://github.com/CS2103JAN2018-F14-B4/main/pull/111[#111]).
+*** Add in encrypt command.
+*** Add in decrypt and setkey command (Pull requests https://github.com/CS2103JAN2018-F14-B4/main/pull/134[#134])
+** Documentation:
+*** Did cosmetic tweaks to existing contents of the User Guide: https://github.com[#14]
+** Community:
+*** PRs reviewed : https://github.com[#12], https://github.com/CS2103JAN2018-F14-B4/main/pull/84[#84], https://github.com/CS2103JAN2018-F14-B4/main/pull/79[#79], https://github.com/CS2103JAN2018-F14-B4/main/pull/68[#68], https://github.com/CS2103JAN2018-F14-B4/main/pull/44[#44]
+** Tools:
+*** Integrated a third party library (Natty) to the project (https://github.com[#42])
```
Comment (takuyakanbr) [inline docs/team/shijin.adoc]: Remove.

File: docs/team/shijin.adoc [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+*** Integrated a new Github plugin (CircleCI) to the team repo
+
+_{you can add/remove categories in the list above}_
```
Comment (takuyakanbr) [inline docs/team/shijin.adoc]: Remove.

File: docs/team/shijin.adoc [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+
+== Contributions to the User Guide
+
+
+|===
+|_Given below are sections I contributed to the User Guide. They showcase my ability to write documentation targeting end-users._
+|===
+
+include::../UserGuide.adoc[tag=edit]
```
Comment (takuyakanbr) [inline docs/team/shijin.adoc]: You didn't write this section...?

File: docs/team/shijin.adoc [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+|===
+|_Given below are sections I contributed to the Developer Guide. They showcase my ability to write technical documentation and the technical depth of my contributions to the project._
+|===
+
+include::../DeveloperGuide.adoc[tag=edit]
+
+include::../DeveloperGuide.adoc[tag=encrypt]
+
+
+== PROJECT: PowerPointLabs
```
Comment (takuyakanbr) [inline docs/team/shijin.adoc]: Remove this line and everything below.

**Pull request merged**